ReactiveCocoa/NSObject+Association.swift

```diff
@@ -1,5 +1,28 @@
 import ReactiveSwift
 
+internal struct AssociationKey<Value> {
+	fileprivate let address: UnsafeMutablePointer<UInt8>
+	fileprivate let `default`: Value!
+
+	init(_ address: UnsafeMutablePointer<UInt8>, default: Value? = nil) {
+		self.address = address
+		self.default = `default`
+	}
+}
+
+internal struct Association {
+	// Byte usage.
+	//
+	// +0: intercepted
+	// +1: signatureCache
+	// +2: selectorCache
+	// +3: runtimeSubclassed
+	// +4: lifetime
+	// +5: lifetimeToken
+	// +6: knownRuntimeSubclass
+	internal static let contiguous = UnsafeMutablePointer<UInt8>.allocate(capacity: 7)
```

Comment:
We should get rid of this and just let the `AssociationKey` allocate a single `UInt8`—without worrying about them being contiguous. I think this method is error prone and an unnecessary optimization. I was concerned about this before, but didn't mention it cause I thought we should split them up, but now it's especially true.